convert2rhel/utils.py

```diff
@@ -560,4 +545,27 @@ def backup(self):
             loggerinst.warning("Can't access %s" % TMP_DIR)
 
 
+def check_readonly_mounts():
+    """
+    Mounting directly to /mnt/ is not in line with Unix FS (https://en.wikipedia.org/wiki/Unix_filesystem).
+    Having /mnt/ and /sys/ read-only causes the installation of the filesystem package to
+    fail (https://bugzilla.redhat.com/show_bug.cgi?id=1887513, https://github.com/oamg/convert2rhel/issues/123).
+    """
+    mounts = get_file_content('/proc/mounts', as_list=True)
+    for line in mounts:
+        _, mount_point, _, flags, _, _ = line.split()
+        flags = flags.split(',')
+        if mount_point not in ('/mnt', '/sys'):
+            continue
+        else:
+            if 'ro' in flags:
+                if mount_point == '/mnt':
+                    loggerinst.critical("/mn/ is read-only - that would cause the conversion to fail.\n"
+                                        "Do not mount directly to /mnt,  mount to its subdirectories instead.")
+                else:  # /sys
+                    loggerinst.critical("/sys is read-only - that would cause the conversion to fail.\n"
+                                        "Make sure it is mounted in read-write mode before executing convert2rhel.")
+            loggerinst.debug("%s is not read-only." % mount_point)
```

Comment:
* Else not necessary for, if-statement continues to next iteration. Keeping the else for the sys critical logger call for readability reasons.
* Changed the text for both log calls

```suggestion
        if 'ro' in flags:
            if mount_point == '/mnt':
                loggerinst.critical("Detected read-only mount to /mnt directory. This will cause the conversion to fail.\n"
                                    "Mount to a subdirectory of /mnt to have read-write.")
            else:  # /sys
                loggerinst.critical("Detected read-only mount to /sys directory. This will cause the conversion to fail.\n"
                                    "Mount in read-write mode before executing convert2rhel.")
        loggerinst.debug("%s mount is not read-only." % mount_point)
```